2-4.js

```diff
@@ -0,0 +1,5 @@
+function arg() {
+  let x = 'true';
+  return( typeof(x) );
+}
+console.log( arg() ); 
```

Comment:
"Write a function for calculating the type of argument and type output to the console"
you should pass some value as function argument and type output to the console
for example function arg('aaaaa')  should output 'string'